yugabyte/run-jepsen.py

```diff
@@ -38,7 +38,7 @@
                         'everything_looks_good'])
 
 # Only for workload, doesn't include test results analysis. Customized for the "set" test.
-SINGLE_TEST_RUN_TIME = 600
+SINGLE_TEST_RUN_TIME = 900
```

Comment:
What's our idea of the desired test runtime?

Reply:
Good question. Since we usually have a few workloads time-out. I thought this would extend the time limit and allow them to finish. But perhaps I am confused about what this parameter does. I'll defere to you on what this value should be.

Reply:
Jepsen tests usually have indefinite running time - they execute as long as we want them to.
This parameter controls that running time, so increasing it would make the test workload itself run longer, thus producing longer history and more YB logs.
Bottom line is, I don't think that increasing it is desirable. While that would potentially allow us to catch issues more frequently, it would give a test more chances to crash, as well as complicate digging through long and heavy logs.
I'd prefer to _lower_ it to 300 - this is the value that used for testing most frequently.

Reply:
Actually, scratch that - we can leave it at 600 for now

Reply:
Thanks for clarifying. Reverted to 600.